src/master/production/sync-replication.md

```diff
@@ -192,21 +192,205 @@ Make sure you already have [ImmuDB installed](../running/download.md).
    Login to immudb
 
    ```shell
-   $ ./immuclient login immudb
+   $ immuclient login immudb
    ```
 
    Select database
 
    ```shell
-   $ ./immuclient use primarydb
+   $ immuclient use primarydb
    ```
 
    Set a value
 
    ```shell
-   $ ./immuclient safeset foo bar
+   $ immuclient safeset foo bar
    ```
 
    The client will block. This is because the primarydb requires 1 sync follower, and since the replica server is down, there is no ack from the replica server, hence synchronous transaction is blocked.
 
 </WrappedSection>
+
+<WrappedSection>
+
+## Recovering from a replica loss
+
+The primary node will continue read and write operations as long as enough replicas can send a write confirmations to the primary node.
+If there are not enough confirmations, write operations will be queued and will wait for enough replicas to synchronize with the cluster.
+Read operations in such case will continue to work.
+
+The simplest way to recover the replica is to simply add another one into the cluster and setup replication in a the same way as during
+the initial cluster setup, e.g.:
+
+```shell
+$ immuadmin database create replicadb -p 3324 \
+   --replication-enabled \
+   --replication-master-address 127.0.0.1 \
+   --replication-master-database primarydb \
+   --replication-follower-username immudb \
+   --replication-follower-password immudb \
+   --replication-master-port 3322 \
+   --replication-sync-enabled \
+   --replication-prefetch-tx-buffer-size 1000 \
+   --replication-commit-concurrency 100
+```
+
+Such replica will start fetching transactions from the primary node and as soon as it synchronizes all transactions
+it will become a valid member of the quorum for transaction confirmation.
+
+### Speeding up initial replica synchronization
+
+The replication process may be slow in case of large databases. Replica will fetch all transactions performing additional checksum
```

Comment:
Replica will fetch all transactions --> The/A Replica will fetch, maybe

Reply:
Rephrased the whole paragraph